File: src/main/java/no/nav/melosys/eessi/models/FagsakRinasakKobling.java
package no.nav.melosys.eessi.models;

import javax.persistence.*;
import lombok.Data;

@Data
@Entity(name = "FAGSAK_RINASAK_KOBLING")
public class FagsakRinasakKobling {

    @Id
    @Column(name = "rina_saksnummer")
    private String rinaSaksnummer;

    @Column(name = "gsak_saksnummer")
    private Long gsakSaksnummer;

    @Enumerated(EnumType.STRING)
    @Column(name = "buc_type")
    private BucType bucType;

}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/FagsakRinasakKobling.kt
package no.nav.melosys.eessi.models

import jakarta.persistence.*

@Entity(name = "FAGSAK_RINASAK_KOBLING")
class FagsakRinasakKobling(
    @Id
    @Column(name = "rina_saksnummer", nullable = false)
    var rinaSaksnummer: String,

    @Column(name = "gsak_saksnummer", nullable = false)
    var gsakSaksnummer: Long,

    @Enumerated(EnumType.STRING)
    @Column(name = "buc_type", nullable = false)
    var bucType: BucType
)
